docs/companion_guide.md

```diff
@@ -411,6 +411,31 @@ billion transactions per second (tps).
 | Schema / Data Representation | JSON Formatted |
 | Query Capabilities | [Not found in documentation] |
 
+| Peergos	   |  https://https://peergos.org/ |
+| -------------- | ---------------------------------------------------------- |
+| Description	   | Peergos is a decentralised protocol and open-source platform for storage, social media and applications |
```

Comment:
You can also run multiple Peergos nodes that sync the same state across one another, enabling the owning entity to secure, manage, and transact their data with others without reliance on location or provider-specific infrastructure

Reply:
@ianopolous there is no direct sync section in this guide, and we have considered putting it in, but haven't yet. If you think sync should be described in some section (i.e description ) please provide a PR with the update. The current description is taken directly off the website https://peergos.org/ @ https://peergos.org/about#:~:text=Peergos%20is%20a%20decentralised%20protocol,end%2Dto%2Dend%20encryption.